server/config/logger.js

```diff
@@ -1,8 +1,36 @@
 const { createLogger, transports, format } = require("winston");
 
-class Logger {
-  constructor() {}
+const getDateFormat = () => {
+  let date = new Date().toDateString();
+  let time = new Date().toLocaleTimeString();
+  return `${date} || ${time}`;
+};
 
+class Logger {
+  constructor() {
```

Comment:
To better lock down our singleton instance, `if Logger.instance` is not null, we should throw an error at the start of the contructor